src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -76,7 +75,9 @@ public class MessageListViewModel : RxConductor<RxScreen>.Collection.AllActive,
 
         public MessageSelectionContext Selection { get; }
 
-        public bool WorkInProgress => workCount > 0 && !Parent.AutoRefresh;
+        public int WorkCount { get; private set; }
```

Comment:
Yep. I thought I had to make it public for the Fody magic to work, but it turns out it works on private properties too. I'll modify the PR.